ngraph/test/backend/interpolate.in.cpp

```diff
@@ -34,29 +34,29 @@ static string s_manifest = "${MANIFEST}";
 
 using TestEngine = test::ENGINE_CLASS_NAME(${BACKEND_NAME});
 
-NGRAPH_TEST(${BACKEND_NAME}, interpolate_down_scales_const_linear)
-{
-    Shape input_shape{1, 1, 2, 4};
-    Shape output_shape{1, 1, 1, 2};
-    op::v0::InterpolateAttrs attrs;
-    attrs.axes = AxisSet{0, 1, 2, 3};
-    attrs.mode = "linear";
-    attrs.align_corners = false;
-    const auto input = make_shared<op::Parameter>(element::f32, input_shape);
-    const auto output_shape_input = op::Constant::create(element::i64, {4}, {1, 1, 1, 2});
-    std::vector<float> intput_data{1.0, 2.0, 3.0, 4.0, 5.0, 6.0, 7.0, 8.0};
-
-    auto interpolate = make_shared<op::Interpolate>(input, output_shape_input, attrs);
-    auto f = make_shared<Function>(interpolate, ParameterVector{input});
-
-    auto backend = runtime::Backend::create("IE_CPU");
-    auto input_tensor = backend->create_tensor(element::f32, input_shape);
-    auto result_tensor = backend->create_tensor(element::f32, output_shape);
-    auto handle = backend->compile(f);
-    copy_data(input_tensor, intput_data);
-
-    handle->call_with_validate({result_tensor}, {input_tensor});
-
-    vector<float> expected_output{1.0f, 2.66666651f};
-    EXPECT_TRUE(test::all_close_f(expected_output, read_vector<float>(result_tensor)));
-}
+// NGRAPH_TEST(${BACKEND_NAME}, interpolate_down_scales_const_linear)
```

Comment:
Why did you comment this test? Maybe we can remove or modify it.

Reply:
Because current CPU implementation of Interpolate satisfies to the Interpolate-1 specification, not Interpolate-4 specification.

Reply:
In this case please use manifest file in order to disable this test for CPU, but don't comment it.

Reply:
How can I do it?

Reply:
Add a  test name to the file: `ngraph/test/runtime/ie/unit_test.manifest`

Reply:
IE_CPU tests disabled in unit_test.manifest: CPU implementation of Interpolate-4 is not merged yet.

Reply:
But if you disabled IE_CPU tests in manifest file, why is it still commented?

Reply:
This test was added as part of reported bug 35806. And it fails for `Interpolate:v0`. Probably it should be changed to use current `Interpolate:v4` in onnx linear mode (and if it pass 35806 probably can be closed).

Reply:
This test is disabled.